src/lib/sw_engine/tvgSwCommon.h

```diff
@@ -235,6 +235,7 @@ struct SwImage
 struct SwBlender
 {
     uint32_t (*join)(uint8_t r, uint8_t g, uint8_t b, uint8_t a);
+    uint32_t (*grayValue)(uint32_t c);
```

Comment:
We don't need this function table in the blender.
tvg won't support multiple formats input.
So.. please call the inline function immediately such as _alpha(), _ialpha()

Reply:
As grayValue (lumaValue) uses color components, this function differs for ARGB and ABGR